pkg/projects/dir.props

```diff
@@ -1,11 +1,123 @@
+
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <!-- This property must be set to the same value as $(PackageOutputPath) for the nuspecs and nupkgs to be binplaced to the intended location. -->
+    <OutputPath>$(PackageOutputPath)</OutputPath>
+
+    <Serviceable>true</Serviceable>
+
+    <IsLineupPackage Condition="'$(PackageTargetRuntime)' == ''">true</IsLineupPackage>
   </PropertyGroup>
+
+  <PropertyGroup Condition="'$(PackageTargetRuntime)' == ''">
+    <SkipValidatePackage>true</SkipValidatePackage>
+    <IncludeRuntimeJson>true</IncludeRuntimeJson>
+  </PropertyGroup>
+
+  <Choose>
+    <When Condition="$(PackageTargetRuntime.StartsWith('win'))">
+      <PropertyGroup>
+        <ApplicationFileExtension>.exe</ApplicationFileExtension>
+        <LibraryFilePrefix></LibraryFilePrefix>
+        <LibraryFileExtension>.dll</LibraryFileExtension>
+        <SymbolFileExtension>.pdb</SymbolFileExtension>
+      </PropertyGroup>
+    </When>
+    <When Condition="$(PackageTargetRuntime.StartsWith('osx'))">
+      <PropertyGroup>
+        <ApplicationFileExtension></ApplicationFileExtension>
+        <LibraryFilePrefix>lib</LibraryFilePrefix>
+        <LibraryFileExtension>.dylib</LibraryFileExtension>
+        <SymbolFileExtension>.dwarf</SymbolFileExtension>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <PropertyGroup>
+        <ApplicationFileExtension></ApplicationFileExtension>
+        <LibraryFilePrefix>lib</LibraryFilePrefix>
+        <LibraryFileExtension>.so</LibraryFileExtension>
+        <SymbolFileExtension>.dbg</SymbolFileExtension>
+      </PropertyGroup>
+    </Otherwise>
+  </Choose>
+
+  <PropertyGroup Condition="'$(PackageRID)' == ''">
+    <PackageRID>$(DistroRid)</PackageRID>
+    <PackageRID Condition="'$(OSEnvironment)' == 'OSX'">osx.10.10-$(Platform)</PackageRID>
+    <PackageRID Condition="'$(OSEnvironment)' == 'Windows_NT'">win10-$(Platform)</PackageRID>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <OfficialBuildRID Include="alpine.3.4.3-x64" />
+    <OfficialBuildRID Include="debian.8-x64" />
+    <OfficialBuildRid Include="debian.8-armel">
+      <Platform>armel</Platform>
+    </OfficialBuildRid>
+    <OfficialBuildRID Include="fedora.23-x64" />
+    <OfficialBuildRID Include="fedora.24-x64" />
+    <OfficialBuildRID Include="linux-x64" />
+    <OfficialBuildRID Include="opensuse.42.1-x64" />
+    <OfficialBuildRID Include="osx.10.10-x64" />
+    <OfficialBuildRID Include="rhel.7-x64" />
+    <OfficialBuildRID Include="ubuntu.14.04-x64" />
+    <OfficialBuildRID Include="ubuntu.16.04-x64" />
+    <OfficialBuildRID Include="ubuntu.16.10-x64" />
+    <OfficialBuildRID Include="win7-x86">
+      <BuidOnRID>win10-x86</BuidOnRID>
+      <Platform>x86</Platform>
+    </OfficialBuildRID>
+    <OfficialBuildRID Include="win7-x64">
+      <BuidOnRID>win10-x64</BuidOnRID>
+    </OfficialBuildRID>
+    <OfficialBuildRID Include="win8-arm">
+      <BuidOnRID>win10-arm</BuidOnRID>
```

Comment:
I didn't change that, it was present in `https://github.com/dotnet/core-setup/blob/master/pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.props#L29`.  I think it's a way of saying to build that `OfficialBuildRid` even if you're on a different rid.  @ericstj